wagtail/snippets/tests/test_viewset.py

```diff
@@ -84,26 +82,28 @@ def setUp(self):
     def test_get_views(self):
         pk = quote(self.object.pk)
         views = [
-            # TODO: Some of these views have been migrated to use the slim_header
-            # only, so there is no header_icon anymore.
-            # ("list", []),
-            # ("add", []),
-            # ("edit", [pk]),
-            # ("delete", [pk]),
-            # ("usage", [pk]),
-            ("unpublish", [pk]),
-            ("workflow_history", [pk]),
-            # ("revisions_revert", [pk, self.revision_1.id]),
-            ("revisions_compare", [pk, self.revision_1.id, self.revision_2.id]),
-            ("revisions_unschedule", [pk, self.revision_2.id]),
+            ("list", [], "headers/slim_header.html"),
+            ("add", [], "headers/slim_header.html"),
+            ("edit", [pk], "headers/slim_header.html"),
+            ("delete", [pk], "header.html"),
+            ("usage", [pk], "headers/slim_header.html"),
+            ("unpublish", [pk], "header.html"),
+            ("workflow_history", [pk], "header.html"),
+            ("revisions_revert", [pk, self.revision_1.id], "headers/slim_header.html"),
+            (
+                "revisions_compare",
+                [pk, self.revision_1.id, self.revision_2.id],
+                "header.html",
+            ),
+            ("revisions_unschedule", [pk, self.revision_2.id], "header.html"),
         ]
-        for view_name, args in views:
+        for view_name, args, header in views:
             with self.subTest(view_name=view_name):
                 response = self.client.get(self.get_url(view_name, args))
                 self.assertEqual(response.status_code, 200)
                 self.assertEqual(response.context["header_icon"], "cog")
                 self.assertContains(response, "icon icon-cog", count=1)
-                self.assertTemplateUsed(response, "wagtailadmin/shared/header.html")
+                self.assertTemplateUsed(response, f"wagtailadmin/shared/{header}")
```

Comment:
I know that we generally shouldn't use formatted strings for template names to ease code searching, but this is test code and not implementation, so if the templates are removed this will fail loudly anyway. I avoided writing the full template names in the `views` cases list so that the diff looks smaller, but I can change this to use the full template paths if you want.

Reply:
Yes for test code I think brevity prevails.